<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LJQcSetupWin</class>
 <widget class="QWidget" name="LJQcSetupWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LJQcSetupWin</string>
  </property>
  <widget class="QPushButton" name="pbBack">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leMcvLow">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>240</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leMchHigh">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>280</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbHct">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>HCT</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbMchc">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>MCHC</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leRbcLow">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbRbc">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>RBC</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leLymPerLow">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leGranNumLow">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbMcv">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>240</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>MCV</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leMchcHigh">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>320</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leMchcLow">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>320</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leGranPerLow">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leMchLow">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>280</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePltLow">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>360</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leWbcLow">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbPlt">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>PLT</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLymPer_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>GRAN%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLymPer_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>GRAN#</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePltHigh">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>360</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leHgbHigh">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leLymNumHigh">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leMcvHigh">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>240</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leHctLow">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>200</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leHctHigh">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>200</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbLymPer">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>LYM%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbMch">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>280</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>MCH</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leRbcHigh">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>160</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leGranPerHigh">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leWbcHigh">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leLymNumLow">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leLymPerHigh">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbWbc">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>WBC</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leGranNumHigh">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leHgbLow">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbLymPer_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>LYM#</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbHgb">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>HGB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSave">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLotNo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Lot No.</string>
   </property>
  </widget>
  <widget class="LineEdit" name="leLotNo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbExpDate">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Exp. Date</string>
   </property>
  </widget>
  <widget class="DateLineEdit" name="leExpDate">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>DateLineEdit</class>
   <extends>LineEdit</extends>
   <header>datelineedit.h</header>
  </customwidget>
  <customwidget>
   <class>DigitLineEdit</class>
   <extends>LineEdit</extends>
   <header>digitlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
